### ISSUE TITLE
Self closing tags cause compile error

### ISSUE THREAD

**Issue by Knovour** [mhanberg/temple#134]
**Describe the bug**
add `hr`, `input` or other self-closing tags in any html.exs files will cause compile error

**To Reproduce**
1. Go to `temple/integration_test/temple_demo/app.html.exs`
2. Add `hr` 
3. See error

```
Compiling 22 files (.ex)

== Compilation error in file lib/temple_demo_web/views/layout_view.ex ==
** (MatchError) no match of right hand side value: {[], []}
    (temple 0.6.1) lib/temple/parser/void_elements_aliases.ex:16: Temple.Parser.VoidElementsAliases.run/1
    (temple 0.6.1) lib/temple/parser.ex:132: Temple.Parser.parse/1
    (elixir 1.12.1) lib/enum.ex:3865: Enum.flat_map_list/2
    (elixir 1.12.1) lib/enum.ex:3866: Enum.flat_map_list/2
    (temple 0.6.1) lib/temple/parser/nonvoid_elements_aliases.ex:26: Temple.Parser.NonvoidElementsAliases.run/1
    (temple 0.6.1) lib/temple/parser.ex:132: Temple.Parser.parse/1
    (temple 0.6.1) lib/temple/parser/nonvoid_elements_aliases.ex:26: Temple.Parser.NonvoidElementsAliases.run/1
    (temple 0.6.1) lib/temple/parser.ex:132: Temple.Parser.parse/1
    (elixir 1.12.1) lib/enum.ex:3865: Enum.flat_map_list/2
    (temple 0.6.1) lib/temple/parser/nonvoid_elements_aliases.ex:26: Temple.Parser.NonvoidElementsAliases.run/1
    (temple 0.6.1) lib/temple/parser.ex:132: Temple.Parser.parse/1
    (elixir 1.12.1) lib/enum.ex:3865: Enum.flat_map_list/2
    (elixir 1.12.1) lib/enum.ex:3866: Enum.flat_map_list/2
    (temple 0.6.1) lib/temple/parser/nonvoid_elements_aliases.ex:26: Temple.Parser.NonvoidElementsAliases.run/1
    (temple 0.6.1) lib/temple/parser.ex:132: Temple.Parser.parse/1
    (temple 0.6.1) lib/temple/engine.ex:41: Temple.Engine.compile/2
    (phoenix 1.5.8) lib/phoenix/template.ex:351: Phoenix.Template.compile/3
    (phoenix 1.5.8) lib/phoenix/template.ex:166: anonymous fn/4 in Phoenix.Template."MACRO-__before_compile__"/2
    (elixir 1.12.1) lib/enum.ex:2356: Enum."-reduce/3-lists^foldl/2-0-"/3
    (phoenix 1.5.8) expanding macro: Phoenix.Template.__before_compile__/1
```

If I add attributes by Map type will cause another error.
```
attrs = %{
  id: "hr"
}
hr attrs
```

Error message:

```
Compiling 1 file (.ex)

== Compilation error in file lib/temple_demo_web/views/layout_view.ex ==
** (FunctionClauseError) no function clause matching in Temple.Parser.Utils.compile_attrs/1

    The following arguments were given to Temple.Parser.Utils.compile_attrs/1:

        # 1
        {:attrs, [line: 44], nil}

    Attempted function clauses (showing 3 out of 3):

        def compile_attrs([])
        def compile_attrs([attrs]) when is_list(attrs)
        def compile_attrs(attrs) when is_list(attrs)

    (temple 0.6.1) lib/temple/parser/utils.ex:10: Temple.Parser.Utils.compile_attrs/1
    (temple 0.6.1) lib/temple/parser/void_elements_aliases.ex:28: Temple.Generator.Temple.Parser.VoidElementsAliases.to_eex/1
    (temple 0.6.1) lib/temple/parser/nonvoid_elements_aliases.ex:38: anonymous fn/2 in Temple.Generator.Temple.Parser.NonvoidElementsAliases.to_eex/1
    (elixir 1.12.1) lib/enum.ex:2356: Enum."-reduce/3-lists^foldl/2-0-"/3
    (temple 0.6.1) lib/temple/parser/nonvoid_elements_aliases.ex:38: Temple.Generator.Temple.Parser.NonvoidElementsAliases.to_eex/1
    (temple 0.6.1) lib/temple/parser/nonvoid_elements_aliases.ex:38: anonymous fn/2 in Temple.Generator.Temple.Parser.NonvoidElementsAliases.to_eex/1
    (elixir 1.12.1) lib/enum.ex:2356: Enum."-reduce/3-lists^foldl/2-0-"/3
    (temple 0.6.1) lib/temple/parser/nonvoid_elements_aliases.ex:38: Temple.Generator.Temple.Parser.NonvoidElementsAliases.to_eex/1
    (elixir 1.12.1) lib/enum.ex:1553: Enum."-map/2-lists^map/1-0-"/2
    (temple 0.6.1) lib/temple/engine.ex:41: Temple.Engine.compile/2
    (phoenix 1.5.8) lib/phoenix/template.ex:351: Phoenix.Template.compile/3
    (phoenix 1.5.8) lib/phoenix/template.ex:166: anonymous fn/4 in Phoenix.Template."MACRO-__before_compile__"/2
    (elixir 1.12.1) lib/enum.ex:2356: Enum."-reduce/3-lists^foldl/2-0-"/3
    (phoenix 1.5.8) expanding macro: Phoenix.Template.__before_compile__/1
    lib/temple_demo_web/views/layout_view.ex:1: TempleDemoWeb.LayoutView (module)
```

It works find when just type `hr id: "hr"`.

**Comment by mhanberg** [comment]
I'm working on a fix for this, hopefully i'll release the patch by tonight

**Comment by mhanberg** [comment]
@Knovour fixed in v0.6.2
